modules/velero-obtaining-by-downloading.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="velero-obtaining-by-downloading_{context}"]
+= Downloading the Velero CLI tool
+
+You can download and install the Velero CLI tool by following the instructions on the link:https://velero.io/docs/v1.8/basic-install/#install-the-cli[Veleo documentation page].
+
+The page includes instructions for:
+
+* MacOS using Homebrew
+* GitHub
+* Windows using Chocolatey
```

Comment:
I think additional linebreak is required here, otherwise it will look like `Windows using Chocolatey .Prerequisites`